kadi/commands/commands_v2.py

```diff
@@ -418,6 +418,7 @@ def update_archive_and_get_cmds_recent(
     cmds_recent.deduplicate_orbit_cmds()
     cmds_recent.remove_not_run_cmds()
     cmds_recent = add_obs_cmds(cmds_recent, pars_dict, rev_pars_dict)
+    cmds_recent.meta["loads_start"] = start.date
```

Comment:
So start is CxoTime(min(loads["cmd_start"])).  Since `loads` there can include loads that are later filtered in the loop that reads into cmds_recent, is that the right time for the logic at 228?  I'd have thought you'd want the minimum time of a from-load cmd in cmds_recent.  But hard to really dig in here.

Reply:
Excellent question which I spent some time considering. The code in the PR should work reliably. The question being asked here is "are there any applicable archive load commands which are NOT available from the recent load commands (i.e. the last 4 weeks of weekly loads)?".  Imagine these inputs:
```
start = Apr-02-2022
Loads = APR0122 (Never run due to anomaly), APR0622 (replan), APR0822,  APR1522, APR2222
cmds_recent.meta["loads_start"] = Apr-01-2022 (first command in APR0122 approx)
```
So what we care about is the first command in the applicable loads in the time span, not the first actually run load command in the span. The reason is that the archive is not going to provide additional commands here because the APR0122 commands will also not be in the archive.

This is a bit subtle (which is never a good thing), so I put this whole response into the code. Does it make sense?

Reply:
Thanks.  That helps.